template/src/ontology/Makefile.jinja2

```diff
@@ -257,12 +262,23 @@ prepare_initial_release: all_assets
 	rsync -R $(RELEASE_ASSETS) $(RELEASEDIR) &&\{% if project.use_dosdps %}
 	mkdir -p $(RELEASEDIR)/patterns && cp -rf $(PATTERN_RELEASE_FILES) $(RELEASEDIR)/patterns &&\{% endif %}
 	rm -f $(patsubst %, ./%, $(CLEANFILES)) &&\
+  {% if project.make_diff %}$(MAKE) $(REPORTDIR)/release-diff.md{% endif %}
```

Comment:
```suggestion
```

Reply:
There wont be a "release" to diff against during `prepare_initial_release` in most cases.

Reply:
ok